### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by feeroz123**
ADded a anew feature .

### CODE REVIEW

File: file3.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 This is file 3 added from Github direclt.y
 Batman Superman
 
+New line ni a new branch
```
Comment (Tkrishna-92) [inline file3.txt:4]: spelling mistake fix it

**Review by Tkrishna-92 — CHANGES_REQUESTED** [review]
reviewed pl make changes

File: file3.txt (hunk 1 above)
Comment (feeroz123) [reply file3.txt]: Implemented

**Review by Tkrishna-92 — APPROVED** [review]
approved

**Pull request merged**